### ISSUE TITLE
Fix image

### ISSUE THREAD

**Issue by gauravumrani** [lwsu/leaflet-polygon-fillPattern#4]
Hi, Is it possible we can have fixed image, for example if we zoom in/out, the image should be fixed, 
I don't want this to happend
![Screenshot 2020-09-26 at 10 07 21 PM](https://user-images.githubusercontent.com/17958457/94345604-b5096500-0044-11eb-8112-6beff7c3ef37.png)
![Screenshot 2020-09-26 at 10 07 31 PM](https://user-images.githubusercontent.com/17958457/94345605-b8045580-0044-11eb-8296-d80641d9a46b.png)

**Comment by nikunj-ncrypted** [comment]
I am looking for same solution. Did you find any?

**Comment by gauravumrani** [comment]
@nikunj-ncrypted yes I was able to solve it, I used this https://github.com/teastman/Leaflet.pattern library for pattern, I don't remember the actual code now as I don't have access.
